src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('RgwBucketListComponent', () => {
   });
 
   beforeEach(() => {
-    rgwBucketService = TestBed.get(RgwBucketService);
+    rgwBucketService = TestBed.inject(RgwBucketService);
```

Comment:
```suggestion
    rgwBucketService = TestBed.get(RgwBucketService);
```

`inject` is new in the Angular 9.

Reply:
Thanks!